e2e/utils.go

```diff
@@ -60,6 +60,8 @@ const (
 	appLabel = "write-data-in-pod"
 
 	noError = ""
+	// labels of rbd pods.
```

Comment:
Mention that this is used for listing/deleting the pods. The `rbdPodLabel` name suggests it is a single label (which could be set on a Pod), not a selector.

Reply:
Done